examples/order-management/client.bal

```diff
@@ -0,0 +1,43 @@
+import ballerina/io;
```

Comment:
shall we add license headers?

Reply:
Added in [9a61dc7](https://github.com/ballerina-platform/module-ballerinax-aws.dynamodbstreams/pull/2/commits/9a61dc79f52588cd3098b3a576772e734b616e93)